README.md

```diff
@@ -35,92 +39,49 @@ Yellow  GND 5  ---       +--+
 
-Open Home Assistant and Add a New ESPHome Integration choosing esphome.  The device may be discovered automatically, in which case just accept the new device and wait a few moments.  A number of sensors will soon appear in `Developer Tools > States`. These will reflect the name of your water heater that you may have configured in the Rheem econet app on a mobile device.  This package though is strictly local and does not require a cloud account or its credentials.
-
-## Protocol Documentation
-
-Example commands to change a heat pump water heater settings [here](https://github.com/stockmopar/esphome-econet/blob/main/m5atom-rs485-econet.yaml)
-
-Decode this into Charcode [here](https://gchq.github.io/CyberChef/#recipe=From_Charcode('Space',16)Strings('Single%20byte',4,'Alphanumeric%20%2B%20punctuation%20(A)',false,false,false/disabled)&input=MHg4MCwgMHgwMCwgMHgxMiwgMHg4MCwgMHgwMCwgMHg4MCwgMHgwMCwgMHgwMywgMHg0MCwgMHgwMCwgMHgxMiwgMHgwMCwgMHgwMCwgMHgxRiwgMHgwMSwgMHgwMSwgMHgwMCwgMHgwNywgMHgwMCwgMHgwMCwgMHg1NywgMHg0OCwgMHg1NCwgMHg1MiwgMHg1MywgMHg0NSwgMHg1NCwgMHg1MCwgMHg0MiwgMHhGOCwgMHgwMCwgMHgwMCwgMHhFNCwgMHhFRQ)
-
-Addresses
-
-Commands
-
-- READ_COMMAND 0x1E (30)
-- ACK 0x06 (6)
-- WRITE_COMMAND 0x1F (31)
-
-Object Strings
-
-| String        | Units         | Gas Tankless | Heat Pump    | Electric | Values     |
-| ------------- | ------------- |------------- |------------- |--------- |------------|
-| WHTRENAB      |               | Y            |Y             |          | 1,0        |
-| WHTRSETP      |               | Y            |Y             |          |            |
-| WHTRMODE      |               | Y            |              |          |            |
-| WHTRCNFG      |               |              |Y             |          | 0,1,2,3,4  |
-| FLOWRATE      |               | Y            |              |          |            |
-| TEMP_OUT      |               | Y            |              |          |            |
-| TEMP__IN      |               | Y            |              |          |            |
-| WTR_USED      |               | Y            |              |          |            |
-| WTR_BTUS      |               | Y            |              |          |            |
-| IGNCYCLS      |               | Y            |              |          |            |
-| BURNTIME      |               | Y            |              |          |            |
-
-Credits:
-
-- <https://github.com/syssi/esphome-solax-x1-mini>
-- <https://github.com/glmnet/esphome_devices>
-- <https://github.com/esphome/esphome/tree/dev/esphome/components/tuya>
+Open Home Assistant and add a new ESPHome Integration, inputing the hostname of your device (e.g. "econet-hpwh.local" by default for Heat Pump Water Heaters).  The device may be discovered automatically, in which case just accept the new device and wait a few moments.  A number of sensors should soon appear in `Developer Tools > States`.
```

Comment:
Remove the mention of dev tools? It's easier to find all sensors in the device itself.